std/conv.d

```diff
@@ -2636,21 +2637,7 @@ do
     assert(parse!uint(str) == 0);
 }
 
-/**
- * Takes a string representing an `enum` type and returns that type.
- *
- * Params:
- *     Target = the `enum` type to convert to
- *     s = the lvalue of the range to _parse
- *
- * Returns:
- *     An `enum` of type `Target`
- *
- * Throws:
- *     A $(LREF ConvException) if type `Target` does not have a member
- *     represented by `s`.
- */
-Target parse(Target, Source)(ref Source s)
+Nullable!(Target) tryParse(Target, Source)(ref Source s)
```

Comment:
Initiative is good but I think we'd need an `Expected` facility to make it right: https://www.youtube.com/watch?v=nVzgkepAg5Y

Reply:
With a `noexcept` Expected facility this would be even better!
We can do this with SumType: https://run.dlang.io/is/82Q0DR

Reply:
I believe we should use a Result type. FYI, @pbackus